.aws/src/iam.ts

```diff
@@ -152,7 +292,7 @@ export class DataFlowsIamRoles extends Construct {
     return {
       actions: ['s3:*Object'],
       effect: 'Allow',
-      resources: [`${this.fileSystem.arn}/data/*`]
+      resources: [`${this.fileSystem.arn}/*`]
```

Comment:
granting task roles access to their appropriate s3 bucket based on deployment type